extensions/dacpac/src/wizard/pages/deployPlanPage.ts

```diff
@@ -98,8 +98,7 @@ export class DeployPlanPage extends DacFxConfigPage {
 			data: this.getColumnData(result),
 			columns: this.getTableColumns(result.dataLossAlerts.size > 0),
 			width: 875,
-			height: 300,
-			ariaRole: 'alert'
```

Comment:
after we remove this - will the loading still be announced as fixed for previous bug?

Reply:
yup. The loader still has ariaRole="alert" so it announces when loading is finished